en/best-practices.md

```diff
@@ -62,9 +73,11 @@ __Examples:__
 | | By default, `stop_name` should not contain generic or redundant words like “Station” or “Stop”, but some edge cases are allowed.<ul><li>When it is actually part of the name (Union Station, Central Station<li>When the `stop_name` is too generic (such as if it is the name of the city). “Station”, “Terminal”, or other words make the meaning clear.</ul> |
 | `stop_lat` & `stop_lon` | Stop locations should be as accurate possible. Stop locations should have an error of __no more__ than four meters when compared to the actual stop position. |
 | | Stop locations should be placed very near to the pedestrian right of way where a passenger will board (i.e. correct side of the street). |
+| | Two stop or station locations should not be closer than would be physically possible in the real world, assuming accurate locations.  If two stops or stations are significantly close, it may indicate incorrect stop or station modelling. |
 | | If a stop location is shared across separate data feeds (i.e. two agencies use exactly the same stop / boarding facility), indicate the stop is shared by using the exact same `stop_lat` and `stop_lon` for both stops. |
 | `parent_station` & `location_type` | Many stations or terminals have multiple boarding facilities (depending on mode, they might be called a bus bay, platform, wharf, gate, or another term). In such cases, feed producers should describe stations, boarding facilities (also called child stops), and their relation. <ul><li>The station or terminal should be defined as a record in `stops.txt` with `location_type = 1`.</li><li>Each boarding facility should be defined as a stop with `location_type = 0`. The `parent_station` field should reference the `stop_id` of the station the boarding facility is in.</li></ul> |
 | | When naming the station and child stops, set names that are well-recognized by riders, and can help riders to identify the station and boarding facility (bus bay, platform, wharf, gate, etc.).<table class='example'><thead><tr><th>Parent Station Name</th><th>Child Stop Name</th></tr></thead><tbody><tr><td>Chicago Union Station</td><td>Chicago Union Station Platform 19</td></tr><tr><td>San Francisco Ferry Building Terminal</td><td>San Francisco Ferry Building Terminal Gate E</td></tr><tr><td>Downtown Transit Center</td><td>Downtown Transit Center Bay B</td></tr></tbody></table> |
+| | If a child stop is significantly far from its parent station or if a child stop is significantly close to a station that is not the stop's parent, it may indicate incorret stop or station modelling. |
```

Comment:
small typo ("incorret" -> "incorrect")

Reply:
typo `far` --> `farther`

Reply:
If the goal is to put what is in the 'old' validator into best practices, what "significantly farther" or "significantly closer" should be defined in the best practices rather than the developers choosing a value.